lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
I agree, I think we should also surround it with `#ifdef INTERPRETER_ASMJS` as well, to not increase non-asmjs interpreter code size which doesn't use DblExtended prefixes.